schema/borrow-schema.py

```diff
@@ -273,7 +274,7 @@ def compare(actual, infra_list, ocds_list, prefix, suffix):
             # Add codes from OCDS.
             reader = csv_reader(f'{ocds_base_url}codelists/documentType.csv')
             for row in reader:
-                if row['Code'] not in seen:
+                if row['Code'] not in seen and row['Code'] != 'finalAudit':
```

Comment:
I tried changing the second condition to `row['Code'] not in ignore`, and that would cause 'contractSchedule' and 'riskProvisions' to also be removed from documentType.csv. Would that be the desired result? If so, I can commit it.

Reply:
Hmmm, I don't recall the reason for removing 'riskProvisions', since it exists in OCDS (not just OCDS for PPPs).

Let's remove 'riskProvisions' from the `ignore` list and change the condition as you suggest.

Reply:
Sounds good. I checked, and 'riskProvisions' has been included since the first commit of the documentType.csv codelist: https://github.com/open-contracting/infrastructure/blame/2202f625476d24c6419c639fe69e11fa6a661221/schema/project-level/codelists/documentType.csv

Reply:
Ah, maybe the older logic was that OCDS for PPPs changed the definition of 'riskProvisions' in a PPP-specific way. I think the same is the case for 'needsAssessment'.